data/core/units.cfg

```diff
@@ -355,7 +355,9 @@ Orcs who were not the strongest of their litter tend to specialize in other skil
         plural_name= _ "race^Ravens"
         description= _ "Ravens are general scavengers, often associating with hunters or predators in the hope of sharing in the kill.  They do not have the powerful beak or claws of the gryphons or raptors, but can still be a threat because of their intelligence and ability to recognize any advantage.
 
-No predators have the numbers to rival orcs or humans, so these races have constant interaction with ravens.  In the aftermath of a battle, the dead and wounded are a banquet for ravens, thus ravens are a nuisance for allies of the defeated but are seen by some victors as part of martial ceremonies.  As intelligent scavengers, ravens do not need war dead to survive.  Some farmers struggle to keep ravens away from freshly sown seeds, but others are happy to have them around as sentries."
+In the aftermath of a battle, the dead and wounded are a banquet for ravens, thus ravens are a nuisance for allies of the defeated but are seen by some victors as part of martial ceremonies.  The violence and large numbers that characterize both orcs and humans bring these races into constant interaction with ravens in this context.  For this reason, the birds have become characters in fables, parables, and superstitions related to war.
```

Comment:
"large numbers" seems more like it should be "large numbers of casualties". However, it feels like too much dwelling on the consequences of violence, ignoring the normal handwave of video games forgetting about the victims.

Reply:
Large numbers could also be the populations in general.  Either interpretation seems OK to me.